src/main/kotlin/com/atlassian/performance/tools/virtualusers/api/config/VirtualUserBehavior.kt

```diff
@@ -18,8 +21,8 @@ class VirtualUserBehavior private constructor(
     internal val diagnosticsLimit: Int,
     internal val browser: Class<out Browser>,
     internal val skipSetup: Boolean,
-    internal val createUsers: Boolean,
```

Comment:
Versioning prowess ranking by me:
* level 1: any SCM, e.g. SVN
* level 2: Git
* level 3: branching
* level 4: atomic commits - full context, [consistent message structure](https://chris.beams.io/posts/git-commit/), single responsibility principle
* level 5: clean commit graph - linear or at least tree-like - rebase, squash, fixup
* level 6: polished diffs - minimal, readable, revertable out-of-order diffs

I never require level 6 prowess, *but* we can easily go for level 6 by replacing the `createUsers: Boolean` with `userGenerator: Class<out UserGenerator>`, which makes the diff smaller, conveys the meaning of the change more directly and doesn't touch `logging` at all

This is absolutely optional for an A+ grade 🙂

Reply:
Thanks for sharing :)